Core/CoreTests/Database/Models/RubricTests.swift

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright (C) 2019-present Instructure, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import XCTest
+@testable import Core
+
+class RubricTests: CoreTestCase {
+
+    override func setUp() {
+
+    }
+
+    func testRubricScope() {
+        let two = Rubric.make(["position": 2, "id": "2"])
+        let one = Rubric.make(["position": 1, "id": "1"])
+        let scope = Rubric.scope(assignmentID: "2")
+
+        let rubrics: [Rubric] = databaseClient.fetch(predicate: scope.predicate, sortDescriptors: scope.order)
+        XCTAssertEqual(rubrics.count, 2)
+        XCTAssertEqual(rubrics.first, one)
+        XCTAssertEqual(rubrics.last, two)
+    }
+
+    func testSaveRating() {
+        var ratings: [RubricRating] = databaseClient.fetch(predicate: NSPredicate.all)
+        databaseClient.delete(ratings)
```

Comment:
Do you have to do this? Doesn't each test get a fresh DB?

Reply:
you're right, setup was causing problems, removing that fixed this right up. Thanks